### ISSUE TITLE
Please append the body parser line in the Example: Express implementation

### ISSUE THREAD

**Issue by RaghavendhraK** [Clever/saml2#100]
In the example, please also add 2 lines to add the body-parser, otherwise the *req.body* after logged in from IDP is always showing empty and thus results in *not a SAML response* error. We almost ended up struggling 2-3 hrs due to this.
```
var saml2 = require('saml2-js');
var fs = require('fs');
var express = require('express');
var app = express();

var bodyParser = require('body-parser');
app.use(bodyParser.urlencoded({
  extended: true
}));
```

Thanks,
Raghu

**Comment by mohit** [comment]
Can you add your change as a PR on `README.md`?

**Comment by RaghavendhraK** [comment]
Please find the PR https://github.com/Clever/saml2/pull/113.

**Comment by cozmo** [comment]
Fixed in #113
